.github/workflows/build.yml

```
@@ -159,17 +159,12 @@ jobs:
     - uses: actions/checkout@v3
     - name: Install Homebrew dependencies
       run: brew install pkg-config openssl@1.1 xz gdbm tcl-tk
-    - name: Prepare Homebrew environment variables
+    - name: Configure CPython with preparing Homebrew environment variables
```

Hm, let's just go with the old step name; all we actually do is configuring the build:

```suggestion
    - name: Configure CPython
```

---

Hey, stop force-pushing, you're messing up my review comments 😆